### PR TITLE
Rename fields to match ECS naming convention + add dashboards

### PR DESCRIPTION
**Pull request by fastlorenzo**
This PR adds ATT&CK Navigator interface that fetches data from Kibana + proposition for name changes

### CODE REVIEW

File: elkserver/install-elkserver.sh [hunk 1]
```diff
@@ -452,18 +459,18 @@ if [ $ERROR -ne 0 ]; then
     echoerror "Could not set the default index for Kibana (Error Code: $ERROR)."
 fi
 
-echo "Installing GeoIP index template adjustment"
-curl -XPUT -H 'Content-Type: application/json' http://localhost:9200/_template/redirtraffic- -d@./templates/elasticsearch-template-geoip-es7x.json >> $LOGFILE 2>&1
```
Comment (fastlorenzo) [inline elkserver/install-elkserver.sh:456]: No longer needed as part of the ES templates

File: elkserver/install-elkserver.sh [hunk 2]
```diff
@@ -452,18 +459,18 @@ if [ $ERROR -ne 0 ]; then
...
+echo "Installing Elasticsearch ILM policy"
+curl -X PUT "http://localhost:9200/_ilm/policy/redelk" -H "Content-Type: application/json" -d @./templates/redelk_elasticsearch_ilm.ndjson; done >> $LOGFILE 2>&1
```
Comment (fastlorenzo) [inline elkserver/install-elkserver.sh:463]: Currently keeps 365 days of data in the index or max 100Go (should be enough for any red team)

File: elkserver/install-elkserver.sh [hunk 3]
```diff
@@ -452,18 +459,18 @@ if [ $ERROR -ne 0 ]; then
...
 ERROR=$?
 if [ $ERROR -ne 0 ]; then
-    echoerror "Could not install GeoIP index template adjust (Error Code: $ERROR)."
+    echoerror "Could not install Elasticsearch ILM policy (Error Code: $ERROR)."
 fi
 
-echo "Setting elasticsearch to single shards"
-curl -XPUT http://localhost:9200/_template/default -H 'Content-Type: application/json' -d '{"index_patterns": ["*"],"order": -1,"settings": {"number_of_shards": "1","number_of_replicas": "0"}}'  >> $LOGFILE 2>&1
```
Comment (fastlorenzo) [inline elkserver/install-elkserver.sh:463]: No longer needed as part of the ES index templates

File: elkserver/logstash/conf.d/20-filter-redir-haproxy_logstash.conf [hunk 4]
```diff
@@ -101,88 +104,103 @@ filter {
...
     }
-    if "ffff" in [redirtraffic.sourceipcdn] {
+    # Apparently never extracted
+    # if "ffff" in [destination][ip] {
+    #   mutate {
+    #     gsub => [
+    #       "[destination][ip]", "\:\:ffff\:", ""
+    #     ]
+    #   }
+    # }
```
Comment (fastlorenzo) [inline elkserver/logstash/conf.d/20-filter-redir-haproxy_logstash.conf:141]: destination.ip field doesn't seem to be extracted anywhere. Commented out and proposition to remove it if not needed.

File: elkserver/scripts/alarm.py [hunk 5]
```diff
@@ -21,21 +22,34 @@ def pprint(r):
...
+        if len(p) > 1:
+            return getValue('.'.join(p[1:]), source[p[0]])
+        else:
+            if p[0] == 'ip':
+                return source[p[0]][0]
+            else:
+                return source[p[0]]
+    else:
+        return None
+
```
Comment (fastlorenzo) [inline elkserver/scripts/alarm.py:37]: This function simplifies getting nested field values (returns None if the field doesn't exist)

File: elkserver/install-elkserver.sh (hunk 3 above)
Comment (MarcOverIP) [reply elkserver/install-elkserver.sh:463]: Great addition!

File: elkserver/install-elkserver.sh (hunk 2 above)
Comment (MarcOverIP) [reply elkserver/install-elkserver.sh:463]: Should be fine

File: elkserver/logstash/conf.d/20-filter-redir-haproxy_logstash.conf (hunk 4 above)
Comment (MarcOverIP) [reply elkserver/logstash/conf.d/20-filter-redir-haproxy_logstash.conf:141]: Could be a simple error, or an artefact of the 0.x days. But it seems to work without it, so Ill remove this.

**Comment by MarcOverIP** [comment]
Great work and major next step. Thanks for the hard work!

**Pull request merged**